Integrations/integration-CounterTack.yml

```diff
@@ -0,0 +1,1695 @@
+
+    # Should we use SSL
+    USE_SSL = not demisto.params().get('unsecure', False)
+    # Service base URL
+    BASE_PATH = '{}/api/v2/'.format(SERVER_URL)
+    # Headers to be sent in requests
+    DEFAULT_HEADERS = {
+        'Content-Type': 'application/json'
+    }
+
+
+    def http_request(method, suffix_url, headers=DEFAULT_HEADERS, body=None):
+        """
+        returns the http request
+
+        """
+        url = BASE_PATH + suffix_url
+        response = requests.request(
+            method,
+            url,
+            auth=(USERNAME, PASSWORD),
+            headers=headers,
+            verify=USE_SSL,
+            data=body
+        )
+        # handle request failure
+        if response.status_code not in {200}:
+            message = parse_error_response(response)
+            return_error('Error in API call to CounterTack with status code {}\n{}'.format(response.status_code, message))
+
+        return response
+
+
+    def parse_error_response(response):
+        try:
+            res = response.json()
+            msg = res.get('message')
+            if res.get('details') is not None and res.get('details')[0].get('message') is not None:
+                msg = msg + "\n"+json.dumps(res.get('details')[0])
+        except:
+            return response.text
+        return msg
+
+
+    """
+
+    ENDPOINTS
+
+    """
+
+
+    def get_endpoints_request():
+        """
+        This request returns a collection of endpoints.
+        """
+        suffix_url = 'endpoints'
+        response = http_request('GET', suffix_url)
+        return response.json()
+
+
+    def get_endpoints():
+
+        """
+        Returns the information on existing endpoints
+        """
+        data = []
+        headers = ['OS', 'Name', 'Threat', 'Status', 'ID', 'IP']
+        endpoints = get_endpoints_request()
+        for endpoint in endpoints:
+            data.append({
+                'ID': endpoint.get('id'),
+                'Name': endpoint.get('name'),
+                'OS': endpoint.get('product_name'),
+                'IP': endpoint.get('ips'),
+                'Status': endpoint.get('status'),
+                'Threat': endpoint.get('threat')
+            })
+            endpoint_standards = {
+                'ID': endpoint.get('id'),
+                'IPAddress': endpoint.get('ips'),
+                'Domain': endpoint.get('domain'),
+                'MACAddress': endpoint.get('mac'),
+                'OS': endpoint.get('product_name'),
+                'OSVersion': endpoint.get('driver_version'),
+                'Model': endpoint.get('current_profile'),
+                'Memory': endpoint.get('memory'),
+                'Processors': endpoint.get('num_cpus')
+            }
+
+        context = {
+            'CounterTack.Endpoint(val.id && val.id === obj.id)': endpoints
+        }
+
+        context = {
+            'Endpoint(val.id && val.id === obj.id)': endpoint_standards
```

Comment:
You have assigned twice context, so the second one will run over the first one.
Make sure to append like you did before that (context[]), atleast the second one